clients/stellar-relay-lib/src/connection/connector/connector.rs

```diff
@@ -23,7 +23,6 @@ pub struct Connector {
 
 	pub(crate) connection_auth: ConnectionAuth,
 	pub(crate) timeout_in_secs: u64,
-	pub(crate) retries: u8,
```

Comment:
removing `retries` field because reconnection is performed indefinitely; unless forced to close.